docs/_articles/en/release-information.md

```diff
@@ -3,9 +3,20 @@ title: Release Information
 lang: en
 ---
 
-### To update the plugin, please follow the linked [instructions](./en/getting-started/install/#upgrade-plug-in).
+### To update Code Analyzer, follow these [instructions](./en/getting-started/install/#upgrade-plug-in).
 <br>
 
+## [v2.13.9](https://github.com/forcedotcom/sfdx-scanner/tree/v2.13.9) (09-14-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v2.13.8...v2.13.9)
+
+### Release Summary
+* NEW: We made some updates to the RetireJS Vulnerability Repository.
```

Comment:
@jfeingold35 So one question: the Quip doc shows "*NEW:* We made some updates to the RetireJS vulnerability database" but this note is "Vulnerability Repository." If "repository" is more accurate, that's fine. But I'd like us to comment on/update the Quip doc so that Quip and the release notes PR match.

Also, I don't believe the vulnerability repository is a brand name? So "vulnerability repository" should be lowercase.